reloadr.py

```diff
@@ -43,6 +43,8 @@ def reload_target(target, kind, filepath=None):
     assert kind in ('class', 'def')
 
     source = get_new_source(target, kind, filepath)
+    decorator = '@autoreload'
+    source = source.replace(decorator, '{}(original=False)'.format(decorator))
```

Comment:
It's unnecessary to watch the same file. Might alter this to just remove the decorator entirely instead